<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Asuszenbook-14" failures="5" tests="9" name="runners.TestRunner" time="42.417" errors="0" timestamp="2025-07-27T20:16:02 IST" skipped="0">
  <testcase classname="runners.TestRunner" name="runScenario" time="4.563"/>
  <testcase classname="runners.TestRunner" name="runScenario" time="4.857">
    <failure message="Expected message to start with: &#039;No special characters allowed&#039;, but got: &#039;Registration was successful. The roll number generated is 6703.&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected message to start with: 'No special characters allowed', but got: 'Registration was successful. The roll number generated is 6703.' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepdefinitions.RegistrationSteps.validate_dynamic_result(RegistrationSteps.java:69)
at ✽.the result should be "No special characters allowed"(file:///C:/Users/Asus/OneDrive/Desktop/Cogni/sprint2/src/test/resources/features/registration.feature:34)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.TestRunner" name="runScenario" time="4.896">
    <failure message="Expected message to start with: &#039;Maximum 30 characters allowed&#039;, but got: &#039;Registration was successful. The roll number generated is 6704.&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected message to start with: 'Maximum 30 characters allowed', but got: 'Registration was successful. The roll number generated is 6704.' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepdefinitions.RegistrationSteps.validate_dynamic_result(RegistrationSteps.java:69)
at ✽.the result should be "Maximum 30 characters allowed"(file:///C:/Users/Asus/OneDrive/Desktop/Cogni/sprint2/src/test/resources/features/registration.feature:40)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.TestRunner" name="runScenario" time="5.727">
    <failure message="Expected message to start with: &#039;Provide value for all fields&#039;, but got: &#039;Provide value for student name, Provide value for mobile number, Provide value for email id, Provide value for cgpa, Provide value for department name, Provide value for backlog count,&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected message to start with: 'Provide value for all fields', but got: 'Provide value for student name, Provide value for mobile number, Provide value for email id, Provide value for cgpa, Provide value for department name, Provide value for backlog count,' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepdefinitions.RegistrationSteps.validate_dynamic_result(RegistrationSteps.java:69)
at ✽.the result should be "Provide value for all fields"(file:///C:/Users/Asus/OneDrive/Desktop/Cogni/sprint2/src/test/resources/features/register_button.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.TestRunner" name="runScenario" time="4.203"/>
  <testcase classname="runners.TestRunner" name="runScenario" time="4.381"/>
  <testcase classname="runners.TestRunner" name="runScenario" time="4.726">
    <failure message="Expected message to start with: &#039;Minimum 2 characters required&#039;, but got: &#039;Registration was successful. The roll number generated is 6700.&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected message to start with: 'Minimum 2 characters required', but got: 'Registration was successful. The roll number generated is 6700.' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepdefinitions.RegistrationSteps.validate_dynamic_result(RegistrationSteps.java:69)
at ✽.the result should be "Minimum 2 characters required"(file:///C:/Users/Asus/OneDrive/Desktop/Cogni/sprint2/src/test/resources/features/registration.feature:16)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.TestRunner" name="runScenario" time="4.356"/>
  <testcase classname="runners.TestRunner" name="runScenario" time="4.708">
    <failure message="Expected message to start with: &#039;No numbers allowed in student name&#039;, but got: &#039;Registration was successful. The roll number generated is 6702.&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected message to start with: 'No numbers allowed in student name', but got: 'Registration was successful. The roll number generated is 6702.' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepdefinitions.RegistrationSteps.validate_dynamic_result(RegistrationSteps.java:69)
at ✽.the result should be "No numbers allowed in student name"(file:///C:/Users/Asus/OneDrive/Desktop/Cogni/sprint2/src/test/resources/features/registration.feature:28)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runners.TestRunner -->
